### PR TITLE
Add common errors to faq

### PR DESCRIPTION
**Pull request by jacoballen1**
This PR is currently a work in progress, feel free to suggest errors to add.

### CODE REVIEW

**Comment by hugmanrique** [comment]
Nice work! Two minor things:
Can you replace the absolute URLs by relative ones? Otherwise, when the site goes live these will need to be changed.
The last error (Velocity forwarding mode) contains a typo (`connecty`)

**Comment by jacoballen1** [comment]
@hugmanrique I will ultimately put the "correct" url in (the one that will be used in the release). But thanks for the tips!

**Comment by hugmanrique** [comment]
There's no need for that, e.g. replace `https://newsite-staging.velocitypowered.com/wiki/users/forwarding/` by `/wiki/users/forwarding/`

File: docs/users/faq.md [hunk 1]
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
...
+
+# Common Errors
+
+``
+Can't connect to server lobby: If you wish to use IP forwarding, 
+please enable it in your Bungeecord config as well! 
+``
+
+``
+Can't conenct to server lobby: Your server did not send a forwarding request to the proxy.
```
Comment (A248) [inline docs/users/faq.md]: connect*

**Comment by jacoballen1** [comment]
Fixed it for both links

File: docs/users/faq.md [hunk 2]
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
...
+These errors are result of improper configuration. See 
+[Player Information Forwarding](/wiki/users/forwarding/) 
+for a proper IP forwarding guide.
+
+``
+Can't connect to server lobby: This server requires you to connect with Velocity
+``
+
+This error is a result of only enabling forwarding on the backend server. To fix this error, 
+ensure that you have set up the correct player information forwarding method on the proxy.
```
Comment (A248) [inline docs/users/faq.md:48]: > correct player information forwarding method

This sounds a bit wordy.

Comment (jacoballen1) [reply docs/users/faq.md:48]: Perhaps "correct forwarding method"?

Comment (A248) [reply docs/users/faq.md:48]: I think "ensure you have set up player info(rmation) forwarding on the proxy" would be sufficient and concise.

Comment (jacoballen1) [reply docs/users/faq.md:48]: well, at that rate we should just leave it how it is... no?
It's not like we have a ton going on for this part of the documentation so we can get away with a little more precision.

File: docs/users/faq.md [hunk 3]
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
 are available. In addition, plugins that support BungeeCord but only require
 that they are installed on the server (nothing on the proxy) typically use
 the BungeeCord plugin messaging channel, which can be supported by BungeeQuack
-for Velocity 1.0.x or natively by Velocity 1.1.0.
\ No newline at end of file
+for Velocity 1.0.x or natively by Velocity 1.1.0.
+
+# Common Errors
+
+``
```
Comment (astei) [inline docs/users/faq.md]: Let's add some headers?

**Review by astei — CHANGES_REQUESTED** [review]

File: docs/users/faq.md (hunk 3 above)
Comment (jacoballen1) [reply docs/users/faq.md]: I have added some headers in the most recent commit. Let me know if that is what you were looking for.

**Comment by jacoballen1** [comment]
I have rebased the branch onto the master to be up to date and condensed the commits into one. I also removed the `package-lock.json` that boosted this pr up to 20k lines.

**Review by astei — APPROVED** [review]
